quickstart/app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Task;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Validator;
+
+class TaskController extends Controller
+{
+
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $tasks = Task::orderBy('created_at', 'asc')->get();
+
+        return view('tasks.index', compact('tasks')
+        );
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create(Request $request)
+    {
+        return view('tasks.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $request->validate([
+            'name' => 'required|max:255',
+        ]);
+        $task = new Task;
+        $task->user_id = Auth::id();
+        $task->name = $request->name;
+
+        $task->save();
+
+        return redirect()->route('task.index')->with('success','Task created successfully.');
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show(Task $task)
+    {
+        return view('tasks.show',compact('task'));
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit(Task $task)
+    {
+        return view('tasks.edit',compact('task'));
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(Request $request, Task $task)
+    {
+        $request->validate([
+            'name' => 'required|max:255',
+        ]);
+
+        $task->update($request->all());
+
+        return redirect()->route('task.index')->with('success','tasks updated successfully');
+    }
+
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy(Task $task)
+    {
+        $task->delete();
+
+        return redirect()->route('task.index')
+                        ->with('success','task deleted successfully');
```

Comment:
```suggestion
        return redirect()
            ->route('task.index')
            ->with('success', trans('task.deleted_successfully'));
```
indent 4 space, i18n, cách sau dấu ,